package.json

```diff
@@ -26,16 +29,9 @@
     "popper.js": "^1.14.5"
   },
   "devDependencies": {
-    "@babel/core": "^7.4.0",
-    "@babel/polyfill": "^7.4.0",
```

Comment:
Since this is frontend we shouldn't just remove polyfills outright. According to https://babeljs.io/docs/babel-polyfill the `@babel/polyfill` package was replaced with `core-js/stable` so we should make sure we have and import that library.

Reply:
Updated

Reply:
> and import that library.

Has this part been done? The babel polyfill got applied automagically by babel at compile time, but you actually have to import the core-js polyfill using an `import` statement. I don't see that done anywhere.

Reply:
Opps forgot that bit, have added